docs/how-to/hipify-clang.rst

```diff
@@ -332,11 +332,6 @@ Print statistics
 
 .. code:: cpp
 
-  [HIPIFY] info: file 'intro.cu' statistics:
-  CONVERTED refs count: 40
-  UNCONVERTED refs count: 0
-  CONVERSION %: 100.0
-  REPLACED bytes: 604

```

Comment:
Please keep it as it is an integral part of the statistics output, which is provided as an example.

Reply:
I can restore it, but it looks like these lines are just duplicates of the lines 340-344 immediately below it?

Reply:
The statistics output format implies having multiple files. If there is a single file to hipify (like in the example provided in the documentation), then the overall statistics and overall file statistics are equal.

Reply:
Removed the file from the PR.